File: cpp/03_005.cpp
#include <iostream>
#include <vector>
#include <iomanip>

void demonstrate_complexity(
        const std::vector<int>& sizes = {10, 100, 1000}) {
    for (int n : sizes) {
        int operations = n * n;
        std::cout << "Size " << n << ": " << std::fixed 
                 << std::setprecision(0) 
                 << std::setw(1) 
                 << std::put_money(operations, false) 
                 << " operations" << std::endl;
    }
}

File: java/03_005.java
import java.text.NumberFormat;
import java.util.Arrays;

public class ComplexityDemo {
  public static void demonstrateComplexity() {
     demonstrateComplexity(new int[]{10, 100, 1000});
  }
  
  public static void demonstrateComplexity(int[] sizes) {
     NumberFormat formatter = NumberFormat.getInstance();
     
     for (int n : sizes) {
        // Calculate quadratic complexity operations
        long operations = (long) n * n;
        System.out.println("Size " + n + ": " + 
           formatter.format(operations) + " operations");
     }
  }
  
  public static void main(String[] args) {
     demonstrateComplexity();
  }
}